src/router/index.js

```diff
@@ -109,37 +132,37 @@ const router = new VueRouter({
   routes
 });
 
-router.beforeEach(loadUser);
-router.beforeEach(checkRoutePermissions);
+// router.beforeEach(loadUser);
+// router.beforeEach(checkRoutePermissions);
 
-export function loadUser (to, from, next) {
-  if (!store.state.currentUser.id) {
-    // TODO: Hook up to route to get user and permissions
-    let currentUser = {
-      firstName: 'Jane',
-      lastName: 'Doe',
-      preferredName: 'Janey',
-      email: 'janedoe@example.com',
-      groups: 'student',
-      id: 20
-    };
+// export function loadUser (to, from, next) {
+//   if (!store.state.currentUser.id) {
+//     // TODO: Hook up to route to get user and permissions
+//     let currentUser = {
+//       firstName: 'Jane',
+//       lastName: 'Doe',
+//       preferredName: 'Janey',
+//       email: 'janedoe@example.com',
+//       groups: 'student',
+//       id: 20
+//     };
 
-    if (!ALLOWED_GROUPS.includes(currentUser.groups)) {
-      currentUser.groups = 'unassigned';
-    }
+//     if (!ALLOWED_GROUPS.includes(currentUser.groups)) {
+//       currentUser.groups = 'unassigned';
+//     }
 
-    store.commit('setCurrentUser', currentUser);
-  }
-  next();
-}
+//     store.commit('setCurrentUser', currentUser);
+//   }
+//   next();
+// }
 
-export function checkRoutePermissions (to, from, next) {
-  if (!to.meta.groups.includes(store.state.currentUser.groups)) {
-    store.commit('setLastURLRequested', to.path);
-    next('/access-denied');
-  } else {
-    next();
-  }
-}
+// export function checkRoutePermissions (to, from, next) {
+//   if (!to.meta.groups.includes(store.state.currentUser.groups)) {
+//     store.commit('setLastURLRequested', to.path);
+//     next('/access-denied');
+//   } else {
+//     next();
+//   }
+// }
```

Comment:
If this comment block is not needed, it can be removed.

Reply:
I left this in since you added it. I thought maybe you wanted it for later.

Reply:
removed it